hack/README.md

```diff
@@ -0,0 +1,70 @@
+docker/hack
+============================
+
+## About
+
+`hack` contains a collection of scripts used to build and manage this
+repository. If there are any issues regarding the intention of a particular
+script (or even part of a certain script), please reach out to us.
+It may help us either refine our current scripts, or add on new ones
+that are appropriate for a given use case.
+
+## DinD (dind.sh)
+
+DinD is a wrapper script which allows Docker to be run inside a Docker
+container. DinD should only be executed inside a Docker
+container with privileged mode enabled.
+
+## Generate Authors (generate-authors.sh)
+
+Generates AUTHORS; a file with all the names and corresponding emails of
+individual Docker contributors. AUTHORS can be found in the home directory of
```

Comment:
Not sure yet where the `hack/` directory is gonna live with the split between "Moby" and "Docker"; perhaps change this to; `individual contributors`